### PR TITLE
docs: fix diffeqarray_to_array function

### PR DESCRIPTION
**Pull request by sathvikbhagavan**
Should fix the doc build as it is currently failing on master - 
https://buildkite.com/julialang/diffeqflux-dot-jl/builds/1518#018d2ff9-e79d-4b7d-8f41-d4554cde4c4c

### CODE REVIEW

File: docs/src/examples/augmented_neural_ode.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ function concentric_sphere(dim, inner_radius_range, outer_radius_range,
         shuffle = true, partial = false)
 end
 
-diffeqarray_to_array(x) = gdev(x).u[1]
+diffeqarray_to_array(x) = gdev(x[1])
```
Comment (ChrisRackauckas) [inline docs/src/examples/augmented_neural_ode.md:39]: This isn't right? It's an AbstractVectorOfArray so you cannot use the scalar indexing.

Comment (sathvikbhagavan) [reply docs/src/examples/augmented_neural_ode.md:39]: Yeah, although it somehow works.

```julia
julia> x1
retcode: Success
Interpolation: 1st order linear
t: 1-element Vector{Float32}:
 1.0
u: 1-element Vector{CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}}:
 [-0.6313359 0.9372761 … -1.0393499 -3.7185428; -0.07272954 0.11323265 … -3.2927783 -1.5652136]

julia> x1[1]
2×4000 CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}:
 -0.631336   0.937276  0.172744   1.22413  -0.856919  -0.479818  …  -1.0803   -0.998729  2.83195   0.797053   1.21714  -1.03935  -3.71854
 -0.0727295  0.113233  0.89953   -1.53696  -0.542933  -0.895658      3.85306   3.58287   1.31141  -3.86188   -3.80056  -3.29278  -1.56521

julia> x1.u[1]
2×4000 CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}:
 -0.631336   0.937276  0.172744   1.22413  -0.856919  -0.479818  …  -1.0803   -0.998729  2.83195   0.797053   1.21714  -1.03935  -3.71854
 -0.0727295  0.113233  0.89953   -1.53696  -0.542933  -0.895658      3.85306   3.58287   1.31141  -3.86188   -3.80056  -3.29278  -1.56521
```

Is the correct way to do is by `x.u[1]`?

Comment (ChrisRackauckas) [reply docs/src/examples/augmented_neural_ode.md:39]: yes, the other way is deprecated.

Comment (ChrisRackauckas) [inline docs/src/examples/augmented_neural_ode.md:39]:
```suggestion
diffeqarray_to_array(x) = gdev(x.u[1])
```

**Review by ChrisRackauckas — APPROVED** [review]

**Pull request merged**